{"name": "test_get_work_item_api_error", "status": "passed", "description": "Test getting a work item when the API call fails.", "attachments": [{"name": "log", "source": "da5cc06e-6024-44cb-94b4-f8b46f1d668a-attachment.txt", "type": "text/plain"}], "start": 1743499571366, "stop": 1743499571366, "uuid": "f9769cfd-4236-42be-966a-91e8b7c48373", "historyId": "52590426741efaa8956082ac7b4c7fa3", "testCaseId": "52590426741efaa8956082ac7b4c7fa3", "fullName": "src.tests.ado.work_item.test_unit_ado_wrapper.TestAdoWorkItemWrapper#test_get_work_item_api_error", "labels": [{"name": "tag", "value": "negative"}, {"name": "tag", "value": "ado"}, {"name": "tag", "value": "unit"}, {"name": "parentSuite", "value": "src.tests.ado.work_item"}, {"name": "suite", "value": "test_unit_ado_wrapper"}, {"name": "subSuite", "value": "TestAdoWorkItemWrapper"}, {"name": "host", "value": "fv-az1120-17"}, {"name": "thread", "value": "2592-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "src.tests.ado.work_item.test_unit_ado_wrapper"}]}